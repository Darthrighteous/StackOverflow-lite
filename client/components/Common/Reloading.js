import React from 'react';

const Reloading = () => (
  <div>RELOADING...</div>
);

export default Reloading;
