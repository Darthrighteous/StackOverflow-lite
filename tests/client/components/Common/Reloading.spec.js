import React from 'react';
import { shallow } from 'enzyme';

import Reloading from '../../../../client/components/Common/Reloading';

describe('Reloading snapshot test', () => {
  test('', () => {
    const reloadingWrapper = shallow(
      <Reloading />
    );
    expect(reloadingWrapper).toMatchSnapshot();
  });
});
