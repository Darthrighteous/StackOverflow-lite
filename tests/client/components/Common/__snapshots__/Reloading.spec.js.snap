// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reloading snapshot test  1`] = `
<div>
  RELOADING...
</div>
`;
